degenotate_lib/opt_parse.py

```diff
@@ -434,6 +446,13 @@ def startProg(globs):
                         "These samples will be used as outgroups in the VCF file and all others as ingroups.");
             # Report VCF outgroup samples
 
+            if globs['vcf-polarized']:
```

Comment:
Maybe I'm missing something, but it looks like we removed the `-p` option above to specify whether the vcf is polarized. Instead it looks as though this is checked later (in `vcf.py`). If that's correct, then this will never be True here so this whole if statement could be removed. But if we want to keep it just in case that's fine.

Reply:
yes, you're right, this one shouldn't be ever executed, I just forgot to remove it after I removed -p flag.
Cleaned it up now.